### PR TITLE
XWIKI-20369: Use correct ellipsis for wysiwyg.admin.fontNames.add.hint

### PR DESCRIPTION
**Pull request by wthrajat**
20 files have been fixed for the issue #20369.

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 1]
```diff
@@ -3613,9 +3613,9 @@ wysiwyg.admin.colorsPerRow.hint=The number of colors to display per row in the c
 wysiwyg.admin.colorPalette.hint=The colors available in the color picker. You can change any color by clicking on it.
 wysiwyg.admin.font=Font settings
 wysiwyg.admin.fontNames.hint=The list of font names available in the font picker. You can add new font names or remove existing ones.
-wysiwyg.admin.fontNames.add.hint=Add font name..
+wysiwyg.admin.fontNames.add.hint=Add font name...
```
Comment (michitux) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties:3616]: An ellipsis is `…`, not `...`, i.e., not three dots but one character. Please also correct this for the other changed messages.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 2]
```diff
@@ -3672,7 +3672,7 @@ extensions.actions.uninstallGlobally=Uninstall from farm
 extensions.actions.upgradeGlobally=Upgrade on farm
 extensions.actions.downgradeGlobally=Downgrade on farm
 extensions.actions.back=Back to list
-extensions.actions.continue=Continue
+extensions.actions.continue=Continue...
```
Comment (michitux) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties:3675]: Why do you think an ellipsis should be added here? This looks totally unrelated to the Jira issue.

**Review by michitux — CHANGES_REQUESTED** [review]
Apart from the comments, please note that our policy is to put the exact Jira title into the commit message, see https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HRule:AlwaysputaJIRAissuereferenceincommitmessages

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties (hunk 1 above)
Comment (wthrajat) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties:3616]: I was about to use ```…``` but I saw 2 dots everywhere and thought why not just add a dot.
Sure I'll fix it :)

**Comment by wthrajat** [comment]
- @michitux Please check the [latest commit](https://github.com/xwiki/xwiki-platform/pull/2058/commits/93ccd0aca349859c3be14c920200a28f56f81a71), where I have replaced all ```...``` with the ellipsis ```…``` in all the properties file.
- I only left dots in the keyword ```a...@domain.org```
- Also the ```Continue...``` is changed to ```Continue```

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 3]
```diff
@@ -3713,7 +3713,7 @@ extensions.uninstall.cleanPages.label=Delete unused wiki pages?
 extensions.uninstall.cleanPages.hint=The following wiki pages are not needed any more so it should be safe to delete them. Unselect the ones that you wish to keep. The wiki pages that have modifications are left unselected so that you don't loose your changes. Select them if those changes are not important.
 extensions.uninstall.cleanPages.selectedCount={0} / {1} pages selected
 extensions.search.submit=Search
-extensions.search.tip=search extension...
+extensions.search.tip=Search extension…
```
Comment (michitux) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties:3716]: Are you sure that the capitalization of this string should be changed? I'm not saying that it is wrong, I'm just asking if you've checked where it is used and that this uppercase letter is really correct in that context.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_cs.properties [hunk 4]
```diff
@@ -2886,7 +2886,7 @@ xe.statistics.module.muchmore=a mnoho dal\u0161\u00EDho!
 xe.webdav.initialize.activex=Nelze inicializovat po\u017Eadovan\u00FD ActiveX objekt.
 xe.webdav.initialize.error=Chyba p\u0159i inicilaizaci editoru share point.
 xe.webdav.install.foxwiki=Tato akce vy\u017Eaduje pro Firefoxinstalaci roz\u0161\u00ED\u0159en\u00ED. M\u00E1 se nainstalovat?
-xe.webdav.error=Ehm....... N\u011Bco se stalo..... Pros\u00EDm zkuste to znovu.
+xe.webdav.error=Ehm……. N\u011Bco se stalo….. Pros\u00EDm zkuste to znovu.
```
Comment (michitux) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_cs.properties:2889]: This looks like an incorrect number of dots/ellipsis characters.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_cs.properties [hunk 5]
```diff
@@ -4829,7 +4829,7 @@ XWiki.XWikiPreferences_javamail_extra_props=Dal\u0161\u00ED vlastnosti Javamail-
 XWiki.XWikiPreferences_admin_email=Email administr\u00E1tora
 ### Missing: XWiki.XWikiPreferences_admin_email.hint=The default email address used to send notification emails from
 ### Missing: XWiki.XWikiPreferences_obfuscateEmailAddresses=Obfuscate Email Addresses
-### Missing: XWiki.XWikiPreferences_obfuscateEmailAddresses.hint=This affects only the email addresses stored in object properties of type Email, as long as the default custom displayer for the Email property type is not overwritten. Example: a...@domain.org
+### Missing: XWiki.XWikiPreferences_obfuscateEmailAddresses.hint=This affects only the email addresses stored in object properties of type Email, as long as the default custom displayer for the Email property type is not overwritten. Example: a…@domain.org
```
Comment (michitux) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_cs.properties:4832]: This replacement is inconsistent with your statement

> I only left dots in the keyword a...@domain.org

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_da.properties [hunk 6]
```diff
@@ -1488,7 +1488,7 @@ core.delete.confirmWithInlinks=Derudover kan sletning af dokumenter ikke fortryd
 ### Missing: core.delete.warningExtensions.tree.freePages=Pages that do not belong to any extension
 ### Missing: core.delete.warningExtensions.tree.selectAll=select all
 ### Missing: core.delete.warningExtensions.tree.selectNone=none
-### Missing: core.delete.warningExtensions.tree.paginationNode={0} more....
+### Missing: core.delete.warningExtensions.tree.paginationNode={0} more….
```
Comment (michitux) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_da.properties:1491]: Wrong number of dots.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_de.properties [hunk 7]
```diff
@@ -4722,7 +4722,7 @@ XWiki.XWikiPreferences_javamail_extra_props=Zus\u00E4tzliche JavaMail-Eigenschaf
 XWiki.XWikiPreferences_admin_email=Admin-E-Mail
 XWiki.XWikiPreferences_admin_email.hint=Die E-Mail Adresse, welche standardm\u00E4\u00DFig als Absender der Benachrichtigungen verwendet wird
 XWiki.XWikiPreferences_obfuscateEmailAddresses=E-Mail-Adressen verschleiern
-XWiki.XWikiPreferences_obfuscateEmailAddresses.hint=Dies betrifft nur E-Mail Adressen, die in speziell als E-Mail markierten Objekteigenschaften abgelegt werden, z. B. im Benutzerprofil, und ist nur dann effektiv, wenn die Standarddarstellung f\u00FCr Objekteigenschaften vom Typ E-Mail nicht \u00FCberschrieben wurde. Beispiel: a...@domain.org
+XWiki.XWikiPreferences_obfuscateEmailAddresses.hint=Dies betrifft nur E-Mail Adressen, die in speziell als E-Mail markierten Objekteigenschaften abgelegt werden, z. B. im Benutzerprofil, und ist nur dann effektiv, wenn die Standarddarstellung f\u00FCr Objekteigenschaften vom Typ E-Mail nicht \u00FCberschrieben wurde. Beispiel: a…@domain.org
```
Comment (michitux) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_de.properties:4725]: This is also not consistent with your statement

> I only left dots in the keyword a...@domain.org

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties [hunk 8]
```diff
@@ -1924,7 +1924,7 @@ rendering.macro.async.parameter.async.description=Activa o desactiva la ejecuci\
 rendering.macro.async.parameter.cached.name=Cacheado
 rendering.macro.async.parameter.cached.description=Habilita o deshabilita el almacenamiento en cach\u00E9 del resultado de la ejecuci\u00F3n de las macros
 rendering.macro.async.parameter.contextEntries.name=Elementos del contexto
-rendering.macro.async.parameter.contextEntries.description=La lista de elementos de contexto necesarios para la ejecuci\u00F3n (wiki, usuario, localizaci\u00F3n, request.base, referencia doc....)
+rendering.macro.async.parameter.contextEntries.description=La lista de elementos de contexto necesarios para la ejecuci\u00F3n (wiki, usuario, localizaci\u00F3n, request.base, referencia doc….)
```
Comment (michitux) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties:1927]: Possibly wrong number of dots (might depend on the language).

**Review by michitux — CHANGES_REQUESTED** [review]
To make this easier to review/possible to review I would suggest to:

* Not change any comments (lines starting with "###").
* Not change any languages we don't speak/we don't know the rules for ellipsis.  You can find some rules at https://en.wikipedia.org/wiki/Ellipsis You may change German and French in addition to English as we can review them or any other languages you speak/know the rules for.

Also see my comments for some additional feedback/values where it definitely went wrong.
